source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -300,11 +301,13 @@ Common::Redis::Client::PoolRequest* InstanceImpl::ThreadLocalPool::makeRequestTo
     if (!absl::SimpleAtoi(ip_port, &ip_port_number) || (ip_port_number > 65535)) {
       return nullptr;
     }
-    try {
-      address_ptr = std::make_shared<Network::Address::Ipv6Instance>(ip_address, ip_port_number);
-    } catch (const EnvoyException&) {
+    Envoy::StatusOr<Network::Address::InstanceConstSharedPtr> error_or_address =
+        Network::Address::InstanceFactory::createInstancePtr<Network::Address::Ipv6Instance>(
+            ip_address, ip_port_number);
+    if (!error_or_address.ok()) {
```

Comment:
The original code might have had a legit reason to swallow this error without a log, so this is probably fine too, FWIW.